modern/base/conan.lock

```diff
@@ -40,13 +40,13 @@
    "4": {
     "ref": "termcap/1.3.1#a75d9f61bf0d1899b64967958dfa6538",
     "options": "fPIC=True\nshared=False",
-    "package_id": "dfbe50feef7f3c6223a476cd5aeadb687084a646",
-    "prev": "383d0c76a636c07aa78d0065f1e0a6f8",
+    "package_id": "76f87539fc90ff313e0b3182641a9bb558a717d2",
+    "prev": "34552acd6624780a5f6251750f40d9c5",
```

Comment:
why package ID has changed?

Reply:
Compiler version, from GCC11 to GCC5.

Reply:
okay, so it's not removed in `package_id`...